app/views/schools/ects/change_email_address_wizard/edit.html.erb

```diff
@@ -0,0 +1,33 @@
+<% page_data(
+    title: "Change email address for #{@wizard.teacher_full_name}",
+    backlink_href: schools_ect_path(@wizard.ect_at_school_period)
+) %>
+
+<p class="govuk-body">
+  Make sure you use an email the ECT can access.
+</p>
+
+<p class="govuk-body">
+  Try to use their school email and do not use a generic email like
+  headteacher@school.com
+</p>
+
+<%= form_with model: @wizard.current_step, url: @wizard.current_step_path do |form| %>
+  <%= content_for(:error_summary) { form.govuk_error_summary } %>
+
+  <%= form.govuk_email_field(
+        :email,
+        label: {text: "What’s the correct email address for #{@wizard.teacher_full_name}"},
+        value: @wizard.current_step.email
```

Comment:
If we go back from the check your answers page, should the email address not be the new one that it was changed to?

This may be one that we need to double check with Jas and card up because I think the name one behaves the same way.